airflow/providers/google/cloud/example_dags/example_cloud_build.py

```diff
@@ -74,41 +101,146 @@
     )
+    # [END howto_operator_create_build_from_repo]
 
+    # [START howto_operator_create_build_from_repo_result]
     create_build_from_repo_result = BashOperator(
-        bash_command="echo '{{ task_instance.xcom_pull('create_build_from_repo')['images'][0] }}'",
+        bash_command="echo '{{ task_instance.xcom_pull('create_build_from_repo')['results'] }}'",
         task_id="create_build_from_repo_result",
     )
+    # [END howto_operator_create_build_from_repo_result]
+
+    # [START howto_operator_list_builds]
+    list_builds = CloudBuildListBuildsOperator(task_id="list_builds", project_id=GCP_PROJECT_ID)
+    # [END howto_operator_list_builds]
+
+    # [START howto_operator_create_build_without_wait]
+    create_build_without_wait = CloudBuildCreateBuildOperator(
+        task_id="create_build_without_wait",
+        project_id=GCP_PROJECT_ID,
+        build=create_build_from_repo_body,
+        wait=False,
+    )
+    # [END howto_operator_create_build_without_wait]
+
+    # [START howto_operator_cancel_build]
+    cancel_build = CloudBuildCancelBuildOperator(
+        task_id="cancel_build",
+        id_="{{ task_instance.xcom_pull('create_build_without_wait')['id'] }}",
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_cancel_build]
+
+    # [START howto_operator_get_build]
+    get_build = CloudBuildGetBuildOperator(
+        task_id="get_build",
+        id_="{{ task_instance.xcom_pull('create_build_without_wait')['id'] }}",
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_get_build]
+
+    # [START howto_operator_retry_build]
+    retry_build = CloudBuildRetryBuildOperator(
+        task_id="retry_build",
+        id_="{{ task_instance.xcom_pull('create_build_without_wait')['id'] }}",
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_retry_build]
 
     # [START howto_operator_gcp_create_build_from_yaml_body]
     create_build_from_file = CloudBuildCreateBuildOperator(
         task_id="create_build_from_file", project_id=GCP_PROJECT_ID,
-        body=str(CURRENT_FOLDER.joinpath('example_cloud_build.yaml')),
+        build=str(CURRENT_FOLDER.joinpath('example_cloud_build.yaml')),
         params={'name': 'Airflow'}
     )
     # [END howto_operator_gcp_create_build_from_yaml_body]
-    create_build_from_storage >> create_build_from_storage_result  # pylint: disable=pointless-statement
 
+    create_build_from_storage >> create_build_from_storage_result  # pylint: disable=pointless-statement
+    create_build_from_storage_result >> list_builds  # pylint: disable=pointless-statement
     create_build_from_repo >> create_build_from_repo_result  # pylint: disable=pointless-statement
+    create_build_from_repo_result >> list_builds  # pylint: disable=pointless-statement
+    list_builds >> create_build_without_wait >> cancel_build  # pylint: disable=pointless-statement
+    cancel_build >> retry_build >> get_build  # pylint: disable=pointless-statement
```

Comment:
Are those disables necessary?

Reply:
No, just ran pylint again and they are exactly not useful at all. I don't know why I put them there.